ephios/core/services/notifications/types.py

```diff
@@ -333,15 +333,21 @@ def send(cls, event: Event, content: str):
                 Notification(
                     slug=cls.slug,
                     user=user,
-                    data=dict(email=participant.email, event_id=event.id, content=content),
+                    data=dict(
+                        email=participant.email,
+                        event_id=event.id,
+                        content=content,
+                        event_title=event.title,
```

Comment:
I must admit this feels like fixing symptoms. There are more occasions in which we will try to fetch objects in Notification sending that have been deleted in the meantime...

Reply:
well usually you won't confirm people directly before deleting the event, so this will be the case where it happens IMO.
What would be your suggestion?

Reply:
I think the best solution would be to send Notifications without delay, but that's for another day.